func_sig_registry/registry/models.py

```diff
@@ -141,3 +150,99 @@ def get_hex_display(self):
 
     def get_bytes_display(self):
         return force_text(self.bytes4_signature)
+
+
+class EventSignature(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True, db_index=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    text_signature = models.TextField(unique=True,
+                                        validators=[
+                                            MinLengthValidator(3), MaxLengthValidator(4096)
+                                        ])
+
+    bytes_signature = models.BinaryField(max_length=32,
+                                            unique=False,
+                                            validators=[MinLengthValidator(32)])
+                                          
+    hex_signature = models.CharField(max_length=64,
+                                        unique=False,
```

Comment:
Same thoughts here.  I would expect we want this field to be unique for the same reasons `bytes_signature` should probably be unique.